tests/python/examples/autodiff/test_simple_derivative.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+
+
+def mkdir_p(dir_path):
```

Comment:
Yea this one should be moved to shared https://github.com/taichi-dev/taichi/blob/master/tests/test_utils.py?

Reply:
Yea you can fix #4322 first :-)